controllers/gcpcluster_controller_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("GCPClusterReconciler", func() {
 
 	Context("Reconcile an GCPCluster", func() {
 		It("should not error and not requeue the request with insufficient set up", func() {
-			ctx := context.Background()
+			ctx := context.TODO()
```

Comment:
This should probably stay `context.Background()` here.
```suggestion
			ctx := context.Background()
```